tectonic/dpx-spc_tpic.c

```diff
@@ -76,7 +76,7 @@ struct spc_tpic_
 
   /* state */
   double     pen_size;
-  int        fill_shape; /* boolean */
+  int        fill_shape; /* bool */
```

Comment:
Fixed. I added an additional commit to change two variables (including this one) from int to bool in this file.